packages/browserify/test/generateConfig.js

```diff
@@ -63,18 +66,19 @@ test('generateConfig - policy ignores global refs', async (t) => {
       const xhr = new window.XMLHttpRequest()
     },
     defaultConfig: false,
-  })
-  const policy = await autoConfigForScenario({ scenario })
-  t.deepEqual(policy, {
-    resources: {
-      one: {
-        globals: {
-          'location.href': true,
-          XMLHttpRequest: true
+    expectedResult: {
+      resources: {
+        one: {
+          globals: {
+            'location.href': true,
+            XMLHttpRequest: true
+          }
         }
       }
     }
-  }, 'policy matches expected')
+  })
+  const policy = await autoConfigForScenario({ scenario })
```

Comment:
dont use checkResult to assert an unrelated value 
just keep the original assertion